packager/media/codecs/aac_audio_specific_config.h

```diff
@@ -84,11 +84,15 @@ class AACAudioSpecificConfig {
   virtual bool Parse(const std::vector<uint8_t>& data);
 
   /// Convert a raw AAC frame into an AAC frame with an ADTS header.
-  /// @param[in,out] buffer contains the raw AAC frame on input, and the
-  ///                converted frame on output if successful; it is untouched
-  ///                on failure.
+  /// @param data the bytes of ADTS header (minus 7 bytes).
+  /// @param data_size the size of a raw AAC frame.
+  /// @param[in,out] audio_frame empty vector on input, and the
```

Comment:
audio_frame is considered as an output only parameter here as we don't use the existing data inside it.

/// @param[out] audio_frame contains the converted frame if successful; it is untouched on failure.